roda-ui/roda-wui/src/main/java/org/roda/wui/api/controllers/BrowserHelper.java

```diff
@@ -461,6 +482,20 @@ public static BrowseFileBundle retrieveBrowseFileBundle(IndexedAIP aip, IndexedR
       bundle.setRepresentationInformationFields(Collections.emptyList());
     }
 
+    ModelService model = RodaCoreFactory.getModelService();
+    DistributedInstance distributedInstance = null;
```

Comment:
You don't need to declare this variable outside the try/catch block,
There is code repetition here too, try to generalize so you don't repeat code.

Reply:
Done